ArduCopter/autoyaw.cpp

```diff
@@ -110,10 +110,10 @@ void Mode::AutoYaw::set_fixed_yaw(float angle_deg, float turn_rate_ds, int8_t di
 
     // calculate final angle as relative to vehicle heading or absolute
     if (relative_angle) {
-        _fixed_yaw_offset_cd = angle_deg * 100.0 * (direction >= 0 ? 1.0 : -1.0);
+        _fixed_yaw_offset_cd = angle_deg * 100.0f * (direction >= 0 ? 1.0 : -1.0);
```

Comment:
should use -fsingle-precision-constant in CPP flags

Reply:
done